### ISSUE TITLE
Hide the path 'json/xxx' from the displayed url

### ISSUE THREAD

**Issue by rjwills28** [dls-controls/machine-status#10]
The application uses the path in the url to load the embedded json file. This means that the displayed urls take the form 'https://xxxxx/json/ms_fe2'. It would be a little less confusing to the user and also neater if we could hide the path from the url.

**Comment by rjwills28** [comment]
I have made a small change to use a MemoryRouter which does not write or read the path from the url. I think this is applicable for both the 'display' and 'web' version. The display version is set to cycle through pages and so there is no need for the back/forward button in the browser. Equally the 'web' version displays tabs to select the view and so these can be used to switch between the views.
See PR: https://github.com/dls-controls/machine-status/pull/11

**Comment by rjwills28** [comment]
I have found an alternative way of changing the displayed URL to something more readable for the machine status screens but still allow you to specify a path to show a certain OPI. Suggestion below:

Monitor display:
- Providing no path: "/" will redirect to the rotation with paths: "/day"," /week", "/fe1", "/fe2", "/message"
- Providing a path such as "/json/ms_day" will display the day view but this will be static (no rotation).

Web display
- Providing no path "/" will redirect to path "/machine-status" and will load the tabbed view to select the view to show.
- Specifying any other path, e.g. "/json/someJsonFile" will load that view.

This has now been implemented and merged in PR11.
